js/project-list.js

```diff
@@ -0,0 +1,58 @@
+/* Project Card */
+
+var ProjectList = {
+  template: _.template($("script#project-list-template").html()),
+  buildHTML: function(projectData) {
+    var id = projectData.id;
+    var timestamp = new Date(projectData.Timestamp);
+    var month = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"][timestamp.getMonth()];
+    var issues = projectData.Issues ? projectData.Issues : "";
+    var issuesAsArray = issues.split(", ");
+    var dataForTemplate = {
```

Comment:
Similarly, this can be simplified as the template doesn't need most of the meta here.